ga4gh/serverconfig.py

```diff
@@ -26,6 +26,12 @@ class BaseConfig(object):
     SIMULATED_BACKEND_VARIANT_DENSITY = 0.5
     SIMULATED_BACKEND_NUM_VARIANT_SETS = 1
 
+    # Options for OIDC
+    # OIDC_PROVIDER = "https://accounts.google.com"
+    # OIDC_CLIENT_ID = "XXXX"
+    # OIDC_CLIENT_SECRET = "XXXX"
+    SERVER_NAME = "localhost:8001"
```

Comment:
This is one of the trickier parts of OIDC.  You have to know exactly what to provide as the callback uri (`redirect_uri` parameter) but this is very, very difficult for a framework to acquire from the webserver reliably, so most frameworks just punt on it.

The way I got around this one is to make sure I'm configuring my OPs from config files, specifically the JSON that the OPs export ([mitre example](https://github.com/Genecloud/simplebeacon/blob/master/oidc_secrets/example-mitreid-connect.json), [google example](https://github.com/Genecloud/simplebeacon/blob/master/oidc_secrets/example-google-identity.json)).  A couple constraints make this possible:

* I'm avoiding dynamic registration (this is suggested by us, but for different reasons)
* I assume that OPs will export JSON documents that include the `redirect_uri` that I've registered, or that I construct those JSON documents myself.

This precludes a "plug and play" experience for users.  I'm torn about this.  OIDC is very hard to get right so it's really important to cover as much ground as possible for the users, but you also don't want to lead them into unsafe practices such as trusting any OP that will assert a claim about a visitors identity.